specs/ChainSpec/Consensus.md

```diff
@@ -1,5 +1,23 @@
 # Consensus
 
+## Definitions and notation
+
+For the purpose of maintaining consensus, transactions are grouped into *blocks*. There is a single preconfigured block \(G\) called *genesis block*. Every block except \(G\) has a link pointing to the *previous block* \(prev(B)\), where \(B\) is the block, and \(G\) is reachable from every block by following those links (that is, there are no cycles).
```

Comment:
I don't think markdown has latex support. Am I wrong?

Reply:
We have MathJax support enabled in [book.toml](https://github.com/nearprotocol/NEPs/blob/master/book.toml). You can look at the generated page here: https://nomicon-pr-57.onrender.com/ChainSpec/Consensus.html.

Reply:
EPIC